Makefile

```diff
@@ -1,3 +1,6 @@
+run:
```

Comment:
Let's remove this as well. Ideally, our entry point should be through python environment with a command line interface rather than going through makefile

Reply:
I was thinking that the Makefile is created for developers. I use it instead of aliases. I will remove it